compositions/README.md

```diff
@@ -38,3 +38,20 @@ Deploys VPC claim resource which uses the above composition.
 ```shell
 kubectl apply -f examples/terrajet-aws-provider/composition-resources/vpc.yaml
 ```
+
+# Configuration Packages
+
+Each folder contains a Crossplane configuration [package](https://crossplane.io/docs/v1.9/concepts/packages.html) definition which bundles all compositions into a single OCI image. 
+
+This is how you build and push the crossplane-aws-blueprints configuration package:
+```shell
+cd aws-provider
+export REPO=<some-oci-repository>
+kubectl crossplane build configuration
+kubectl crossplane push configuration $REPO/crossplane-aws-blueprints:v0.0.1
+```
+
+And this is how you would install that package on to your control plane:
+```shell
+kubectl crossplane install configuration $REPO/crossplane-aws-blueprints:v0.0.1
+```
```

Comment:
Can you include a declarative example for this?

Reply:
done https://github.com/aws-samples/crossplane-aws-blueprints/pull/51/commits/d3ea9ecc8858f271e5a8dba1bdb0f6f92c98c38e